woocommerce/woocommerce-paypal-payments
# Filter PayPal-supported language codes

### Description
When selecting a language in WordPress like German [formal] or Norsk [Bokmål] banners and buttons do not appear on the selected pages and also there will be an error after trying to buy a product on the checkout page.

### Steps to test:
1. Select German [formal] in WordPress general settings.
2. Enable all banners and buttons in the plugin settings and save it.
3. Check the product pages.
4. Buy a product.
